src/Components/Context/AuthContext/AuthContext.tsx

```diff
@@ -7,15 +7,14 @@ export default function AuthContextProvider(props) {
   // ******************* to baseUrl for apis *******************
   const baseUrl = `http://upskilling-egypt.com:3000/api/v0`;
```

Comment:
The URL and request headers should be in a constants files and used in the application as needed, instead of having them in a context.